source/robot_model/README.md

```diff
@@ -0,0 +1,85 @@
+# robot_model
+
+The `robot_model` library is a wrapper for the [Pinocchio](https://github.com/stack-of-tasks/pinocchio) library, used
+to compute dynamics model of a robot.
+It is specifically designed to work seamlessly with the `state_representation` library, offering conversion functions
+between the states and input/output needed for `pinocchio`.
+
+The library offers a single class `Model` that encapsulate most of the functionalities from `pinocchio`, as well
+as extra functions.
+A `Model` is defined by a robot name, and initialized from a `URDF` description file of the robot.
+
+```cpp
+robot_model::Model model("myrobot", "path/to/urdf/myrobot.urdf")
+```
+
+At creation, it uses the parser from `pinocchio` to read the `URDF` file and fill the needed parameters for computation.
+Most common robotics functionalities are implemented such `forward_geometry`, `inverse_geometry`, `forward_kinematic`,
```

Comment:
One of them is not implemented yet right? :smile:

Reply:
well @patr3 is working on that